lib/plug/adapters/test/conn.ex

```diff
@@ -119,6 +119,16 @@ defmodule Plug.Adapters.Test.Conn do
     :ok
   end
 
+  def upgrade(%{owner: owner, ref: ref}, :unsupported = protocol, opts) do
```

Comment:
Maybe call it `:not_supported` for consistency?

Reply:
Done